docs/howto/instrument-your-charm-with-tracing-telemetry.md

```diff
@@ -0,0 +1,146 @@
+(how-to-instrument-your-charm-with-tracing-telemetry)=
+# How to instrument your charm with tracing telemetry
+
+```{note}
+
+This howto has been incorporated into [the Tempo charm docs](https://charmhub.io/topics/charmed-tempo-ha/instrument%20your%20charm).
+
+```
```

Comment:
Does this mean that we should actually remove this page?

Reply:
:+1: tracked this in my actions doc